### DISCUSSION TITLE
https://demonkingswarn.is-a.dev/blog/2024/04/13/fuck-ubisoft

### DISCUSSION THREAD

**Discussion by giscus** [DemonKingSwarn/blog#6] [General]
# https://demonkingswarn.is-a.dev/blog/2024/04/13/fuck-ubisoft

https://demonkingswarn.is-a.dev/blog/2024/04/13/fuck-ubisoft

**Comment by 4cecoder** [comment]
### Ubisoft: the ultimate cuck and total a-hole, taking away our games and demanding more money. Time to demand better treatment from our favorite game companies!

Ubisoft has lost their marbles. They think it's cool to take away games you bought, like a bully stealing your lunch money. And then they want more money for new games? Talk about being tone-deaf! It's time we all rally together and demand better treatment from our favorite game companies – let's show them that treating customers fairly is the key to success in this ever-evolving industry!

Ubisoft is acting like an OnlyFans chick who expects you to pay for every little thing – from buying them luxary bags to access their new content – and then has the nerve to call you a "pay pig" when you express your dissatisfaction. It's like Uncle Sam calling you a piggy bank, demanding more and more money while dominating your wallet with their tax scam.
